Day7n8n9n10/github-api-project-ts/src/containers/Header/Header.tsx

```diff
@@ -2,12 +2,14 @@ import React from "react";
 
 import classes from "./header.styles.module.css";
 import ClassBasedComponent1 from "../../miscellaneous/ParentClassBasedComponent";
+import { useThemeContext } from "../../contexts/ThemeContext";
 
 type Props = {};
 
 const Header = (props: Props) => {
+  const {mode}=useThemeContext()
```

Comment:
i was using in my code, something like this  const { color } = useContext(ThemeContext);  //was passing the Themecontext to useContext,
 does **const {mode}=useThemeContext() ** -> also do the same thing?

Reply:
It's the same but is useful when you're using global contexts, you'll reduce one less import and don't have to write everytime ```const { color } = useContext(ThemeContext)```.
I've exported it like `export const useThemeContext = useContext(ThemeContext)`